<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DTableData</class>
 <widget class="QDialog" name="DTableData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Material Library</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="in_row">
       <property name="text">
        <string>N_row: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="si_row">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="in_col">
       <property name="text">
        <string>N_column:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="si_col">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="w_tab"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="b_close">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../Resources/pyleecan.qrc"/>
  <include location="../../Resources/pyleecan.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>b_close</sender>
   <signal>accepted()</signal>
   <receiver>DTableData</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>b_close</sender>
   <signal>rejected()</signal>
   <receiver>DTableData</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
